dashboard/lib/build_dashboard_page.dart

```diff
@@ -111,7 +111,7 @@ class BuildDashboardPageState extends State<BuildDashboardPage> {
           child: FocusTraversalGroup(
             child: SizedBox(
               width: 500,
-              height: 350,
+              height: 360,
```

Comment:
Based on the goldens, it seems like we should add some extra height. Can you confirm it looks fine?

Reply:
I do confirm the 10px are needed, or the the bottom buttons will be cut off. I confirm this looks fine.